public/scripts/main/services/loadService.js

```diff
@@ -117,39 +117,92 @@ coaxsApp.service('loadService', function ($http, targetService, supportService)
     $http.get('/geojson/pois')
     .success(function (data, status) {
       if (status == 200) {
+       
         var circles = [];
         var poiUsers = [];
+        var homeIcons = [];
+
+        // var iconColor = ('00000'+(Math.random()*(1<<24)|0).toString(16)).slice(-6)
 
         var smallMarkerOptions = {
-          radius      : 5,
-          fillColor   : 'rgba(139,139,210,0.3)',
-          color       : 'rgba(36,36,76,0.8)',
+          radius      : 10,
+          fillColor   : 'yellow',
```

Comment:
should we specific the yellow i think this is a color that does not have std hex interpretation